src/containers/UploadContainer.js

```diff
@@ -51,44 +57,21 @@ const UploadContainer = props => {
       });
   };
 
-  // Upload a file when a document is selected from the browser window
-  const handleSelectedFile = (e, createFile) => {
-    e.preventDefault();
-    uploadAndAnnotate(createFile, e.target.files[0]);
-  };
-
-  // Upload a file when it is dropped on the container
-  const handleDrop = (e, createFile) => {
-    e.preventDefault();
-    setDragging(false);
-
-    if (e.dataTransfer.files && e.dataTransfer.files.length > 0) {
-      uploadAndAnnotate(createFile, e.dataTransfer.files[0]);
-    }
-  };
-
   return (
-    <Mutation
-      mutation={CREATE_FILE}
-      refetchQueries={res => [
-        {
-          query: GET_STUDY_BY_ID,
-          variables: {kfId: props.match.params.kfId},
-        },
-      ]}
-    >
-      {(createFile, {data, error}) => (
-        <FileUploadTarget
-          error={error}
-          dragging={dragging}
-          handleDragOver={handleDragOver}
-          onDragEnter={handleDragEnter}
-          onDragLeave={handleDragLeave}
-          handleDrop={e => handleDrop(e, createFile)}
-          handleSelectedFile={e => handleSelectedFile(e, createFile)}
-          instructions="To upload files, drag and drop them here"
-        />
-      )}
+    <Mutation mutation={CREATE_FILE} refetchQueries={res => refetchAfterUpload}>
+      {(createFile, {data, error}) =>
+        children
```

Comment:
Since you only need to render if there are children, you can use {children && (<div></div>)}` instead of the ternary.